### PR TITLE
Make browser.html usable on electron.

### PR DESCRIPTION
**Pull request by Gozala**
With this changes you can run browser.html on top of Electron. To do so you need, to:

1. Install electron on your machine. You can do it by running `npm install -g electron-prebuilt`
2. Run browser.html on top of Electron from project root via command `electron .`

### CODE REVIEW

File: src/browser/web-view/electron-frame.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  ( event ) =>
+  ( { type: "LoadFail"
+    , time: performance.now()
+    , reason: event.errorDescription
+    , code: event.errorCode
+    }
+  );
+
+// @TODO: We are missing "Connected" event and I think 'did-get-response-details'
+// maybe it in electron.
```
Comment (paulrouget) [inline src/browser/web-view/electron-frame.js]: It is.

File: src/common/keyboard.js [hunk 2]
```diff
@@ -13,6 +13,94 @@ import * as OS from './os';
...
+      const key =
+        ( charCode === 13
+        ? keyTable[charCode]
+        : String.fromCharCode(charCode)
+        );
+
+      return key
+    }
+    else if (this.type === 'keyup' || this.type === 'keydown') {
+      return keyTable[this.keyCode]
```
Comment (paulrouget) [inline src/common/keyboard.js]: We should never use `keyCode`. What is that for?

Comment (Gozala) [inline src/common/keyboard.js]: > We should never use keyCode. What is that for?

Webkit does not yet implement `KeyboardEvent.prototype.key` & this is a polyfill for Electron to handle keybindings, only defined if `KeyboardEvent.prototype.key` is not defined.

**Comment by paulrouget** [comment]
r+

**Pull request merged**